themes/theme-gmd/components/AppBar/presets/DefaultBar/index.jsx

```diff
@@ -138,31 +123,6 @@ class AppBarDefault extends PureComponent {
     }
   }
 
-  /**
-   * Removes the event listeners when the component unmounts.
-   */
-  componentWillUnmount() {
-    UIEvents.removeListener(NavDrawer.EVENT_CLOSE, this.setFocus);
-  }
-
-  /**
-   * Sets the a11y focus to the first focusable element within the AppBar.
-   */
-  setFocus = () => {
-    const { target } = this.state;
-
-    if (!target) {
-      return;
-    }
-
-    // Set the focus to the first focusable element for screen readers.
-    const focusable = target.querySelector('button:not([aria-hidden="true"]), [tabindex]:not([tabindex="-1"])');
```

Comment:
as discussed i'd like to keep the old approach with some modifications. not every app bar has a title e.g. the browse and more pages on the ios theme. for these pages the focus stays on the just pressed tab bar icon.

i'd recommend to put a marker e.g. a human readable class to the app bar title component and search for this instead of the pressable buttons. if not found you can fallback to something else - maybe the elements that are selected in the old code.

please  also put pack the  event listeners in this version of the DefaultBar since they handle setting of the focus when the navdrawer closes.